blocks/newsletter-subscribe/newsletter-subscribe.css

```diff
@@ -0,0 +1,371 @@
+  display: none;
+  height: 23px;
+  font-size: 13px;
+  font-family: "Times New Roman", Times, serif;
+  color: #e76468;
+  font-weight: 700;
+  line-height: 15px;
+  padding-bottom: 8px;
+  text-align: center;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input.invalid + .invalid-email-text {
+  display: block;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-to-all-wrapper {
+  max-width: 550px;
+  width: 100%;
+  margin: 0 auto 60px;
+  padding-top: 7px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-to-all-wrapper .toggle-label {
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 22px;
+  letter-spacing: 0.3em;
+  float: left;
+  color: #151517;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item {
+  display: flex;
+  background-color: #fff;
+  gap: 25px;
+  padding-right: 31px;
+  height: 203px;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .image-wrapper {
+  width: 285px;
+  height: 203px;
+  min-width: 285px;
+  min-height: 203px;
+  overflow: hidden;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .image-wrapper picture,
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .image-wrapper img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .text-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  margin: 25px 0 5px;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .text-wrapper .item-title {
+  font-family: var(--font-tungsten);
+  font-size: 42px;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0.05em;
+  text-align: left;
+  text-transform: uppercase;
+  color: #000;
+  height: 44px;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .text-wrapper p {
+  margin: 0;
+  font-size: 14px;
+  font-weight: 325;
+  line-height: 23px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #000;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.newsletter-subscribe.block .subscribe-form .subscribe-list .subscribe-item .checkbox-wrapper {
+  margin-top: 33px;
+  margin-left: auto;
+}
+
+.newsletter-subscribe.block .subscribe-form .form-footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.newsletter-subscribe.block .subscribe-form .form-footer .sign-up-button {
+  padding: 0;
+  border: none;
+  color: white;
+  text-transform: uppercase;
+  background-color: #ed3e49;
```

Comment:
I created https://github.com/headwirecom/helix-sportsmagazine/issues/214 to clean up CSS colors.